packages/bls-kms-local/src/agent/BlsKeyManagementSystem.ts

```diff
@@ -0,0 +1,100 @@
+import { blsSign, generateBls12381G2KeyPair } from '@mattrglobal/node-bbs-signatures'
+import Debug from 'debug'
+
+import { IKey, ManagedKeyInfo, MinimalImportableKey, TKeyType } from '@veramo/core'
+import { AbstractPrivateKeyStore, ManagedPrivateKey } from '@veramo/key-manager'
+import { KeyManagementSystem } from '@veramo/kms-local'
+import { BlsManagedKeyInfoArgs, KeyType } from '../types/IBlsKeyManager'
+
+const debug = Debug('veramo:kms:bls:local')
+
+export class BlsKeyManagementSystem extends KeyManagementSystem {
+  private readonly privateKeyStore: AbstractPrivateKeyStore
+
+  constructor(keyStore: AbstractPrivateKeyStore) {
+    super(keyStore)
+    this.privateKeyStore = keyStore
+  }
+
+  async importKey(args: Exclude<MinimalImportableKey, 'kms'>): Promise<ManagedKeyInfo> {
+    if (args.type === KeyType.Bls12381G2) {
```

Comment:
Change around the if / super flow using an early return to make it more clear that by default we call the super method

Reply:
Fixed!